frontend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Should we add gitignore lines for IDE files and directories? I'm thinking VS code, any suggestions?

Reply:
That would be good. But probably more important on a project wide basis rather than just within the frontend folder

Reply:
I was expecting that we'd open the project in the root of the frontend folder instead of the root of the repo, in which case it'd be safe to add it to this .gitignore.